src/gcovr/formats/sonarqube/write.py

```diff
@@ -44,10 +46,18 @@ def write_report(
                 line_node.set("lineNumber", str(linecov.lineno))
                 line_node.set("covered", "true" if linecov.is_covered else "false")
 
-                if linecov.branches:
-                    stat = linecov.branch_coverage()
-                    line_node.set("branchesToCover", str(stat.total))
-                    line_node.set("coveredBranches", str(stat.covered))
+                if options.sonarqube_metric != "line":
```

Comment:
I'm unsure with the option to only have line coverage. Do you think that this is required?